Copy employees to customer table example job fails

When running the `Copy employees to customer table.analysis.xml` example job, which is shipped with DataCleaner by default, the job fails to finish and some stacktraces are logged in the execution log:

```
2:01:10ERROR: An error occurred in the component: orderdb - CUSTOMERS (18 columns)

Could not execute batch: org.hsqldb.jdbc.jdbcPreparedStatement@3f272cb3[sql=[INSERT INTO PUBLIC."CUSTOMERS" (CUSTOMERNUMBER,CUSTOMERNAME,CONTACTLASTNAME,CONTACTFIRSTNAME,PHONE,ADDRESSLINE1,CITY,COUNTRY,SALESREPEMPLOYEENUMBER) VALUES (?,?,?,?,?,?,?,?,?)], parameters=[[5012], [My company], [Vanauf], [George], [My phone no], [My address], [My city], [My country], [1323]]]: failed batch

org.apache.metamodel.MetaModelException: Could not execute batch: org.hsqldb.jdbc.jdbcPreparedStatement@3f272cb3[sql=[INSERT INTO PUBLIC."CUSTOMERS" (CUSTOMERNUMBER,CUSTOMERNAME,CONTACTLASTNAME,CONTACTFIRSTNAME,PHONE,ADDRESSLINE1,CITY,COUNTRY,SALESREPEMPLOYEENUMBER) VALUES (?,?,?,?,?,?,?,?,?)], parameters=[[5012], [My company], [Vanauf], [George], [My phone no], [My address], [My city], [My country], [1323]]]: failed batch
	at org.apache.metamodel.jdbc.JdbcUtils.wrapException(JdbcUtils.java:70)
	at org.apache.metamodel.jdbc.JdbcBatchUpdateCallback.closePreparedStatement(JdbcBatchUpdateCallback.java:64)
	at org.apache.metamodel.jdbc.JdbcUpdateCallback.close(JdbcUpdateCallback.java:80)
	at org.apache.metamodel.jdbc.JdbcDataContext.executeUpdate(JdbcDataContext.java:842)
	at org.datacleaner.beans.writers.InsertIntoTableAnalyzer.run(InsertIntoTableAnalyzer.java:414)
	at org.datacleaner.beans.writers.InsertIntoTableAnalyzer.run(InsertIntoTableAnalyzer.java:76)
	at org.datacleaner.util.WriteBuffer.flushBuffer(WriteBuffer.java:84)
	at org.datacleaner.util.WriteBuffer.addToBuffer(WriteBuffer.java:60)
	at org.datacleaner.beans.writers.InsertIntoTableAnalyzer.run(InsertIntoTableAnalyzer.java:377)
	at org.datacleaner.job.runner.AnalyzerConsumer.consumeInternal(AnalyzerConsumer.java:71)
	at org.datacleaner.job.runner.AbstractRowProcessingConsumer.consume(AbstractRowProcessingConsumer.java:159)
	at org.datacleaner.job.runner.ConsumeRowHandlerDelegate.consume(ConsumeRowHandlerDelegate.java:64)
	at org.datacleaner.job.runner.ConsumeRowHandlerDelegate.processNext(ConsumeRowHandlerDelegate.java:90)
	at org.datacleaner.job.runner.TransformerConsumer.consumeInternal(TransformerConsumer.java:113)
	at org.datacleaner.job.runner.AbstractRowProcessingConsumer.consume(AbstractRowProcessingConsumer.java:159)
	at org.datacleaner.job.runner.ConsumeRowHandlerDelegate.consume(ConsumeRowHandlerDelegate.java:64)
	at org.datacleaner.job.runner.ConsumeRowHandler.consumeRow(ConsumeRowHandler.java:146)
	at org.datacleaner.job.tasks.ConsumeRowTask.execute(ConsumeRowTask.java:51)
	at org.datacleaner.job.concurrent.TaskRunnable.run(TaskRunnable.java:61)
	at org.datacleaner.job.concurrent.MultiThreadedTaskRunner.assistExecution(MultiThreadedTaskRunner.java:129)
	at org.datacleaner.job.runner.ErrorAwareTaskRunnerWrapper.assistExecution(ErrorAwareTaskRunnerWrapper.java:95)
	at org.datacleaner.job.runner.RowConsumerTaskListener.awaitTasks(RowConsumerTaskListener.java:73)
	at org.datacleaner.job.runner.SourceTableRowProcessingPublisher.processRowsInternal(SourceTableRowProcessingPublisher.java:244)
	at org.datacleaner.job.runner.AbstractRowProcessingPublisher.processRows(AbstractRowProcessingPublisher.java:208)
	at org.datacleaner.job.tasks.RunRowProcessingPublisherTask.execute(RunRowProcessingPublisherTask.java:43)
	at org.datacleaner.job.concurrent.TaskRunnable.run(TaskRunnable.java:61)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.sql.BatchUpdateException: failed batch
	at org.hsqldb.jdbc.jdbcStatement.executeBatch(Unknown Source)
	at org.hsqldb.jdbc.jdbcPreparedStatement.executeBatch(Unknown Source)
	at org.apache.commons.dbcp.DelegatingStatement.executeBatch(DelegatingStatement.java:297)
	at org.apache.commons.dbcp.DelegatingStatement.executeBatch(DelegatingStatement.java:297)
	at org.apache.metamodel.jdbc.JdbcBatchUpdateCallback.closePreparedStatement(JdbcBatchUpdateCallback.java:45)
	... 27 more
```

--- a/desktop/ui/src/main/resources/datacleaner-home/jobs/Copy employees to customer table.analysis.xml
+++ b/desktop/ui/src/main/resources/datacleaner-home/jobs/Copy employees to customer table.analysis.xml
@@ -34,7 +34,7 @@
             <descriptor ref="Insert into table"/>
             <properties>
                 <property name="Buffer size" value="TINY"/>
-                <property name="Column names" value="[SALESREPEMPLOYEENUMBER,CONTACTLASTNAME,CONTACTFIRSTNAME,CUSTOMERNUMBER,CUSTOMERNAME,PHONE,ADDRESSLINE1,CITY,COUNTRY]"/>
+                <property name="Column names" value="[SALESREPEMPLOYEENUMBER,CONTACTLASTNAME,CONTACTFIRSTNAME,CUSTOMERNUMBER,CUSTOMERNAME,PHONE,ADDRESSLINE1,CITY,COUNTRY,JOBTITLE]"/>
                 <property name="Datastore" value="orderdb"/>
                 <property name="Error log file location" value="C:/Users/kasper/AppData/Local/Temp"/>
                 <property name="How to handle insertion errors?" value="STOP_JOB"/>
@@ -60,6 +60,7 @@
             <input value="My address" name="Values"/>
             <input value="My city" name="Values"/>
             <input value="My country" name="Values"/>
+            <input ref="col_7" name="Values"/>
         </analyzer>
     </analysis>
 </job>

